apm-application-toolkit/apm-toolkit-micrometer-registry/src/main/java/org/apache/skywalking/apm/meter/micrometer/SkywalkingDistributionSummary.java

```diff
@@ -47,7 +47,7 @@
     private final Counter sum;
 
     /**
-     * Max amount in this summary
+     * MaxAmount in this summary
```

Comment:
What does this mean?

Reply:
I think it's a little clearer

Reply:
But your changes are using two different styles.

Reply:
Sorry,I didn't catch your meaning

Reply:
You can change it to whatever you feel is more appropriate

Reply:
Personally, I prefer don't change this one.

Reply:
All right, let me undo it